### PR TITLE
use prebuilt ci build image

### PR DESCRIPTION
**Pull request by danielnorberg**

### CODE REVIEW

**Comment by frsyuki** [comment]
nice improvements 👍

File: circle.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ machine:
     version: oraclejdk8
   environment:
     TERM: dumb  # don't let gradle use fancy ansi seq code
+    BUILD_IMAGE: danieln/digdag-build:20160823085939Z-4c3c94d6925e4e476fca04260ec6be6352351de2
```
Comment (frsyuki) [inline circle.yml]: this image is built using `docker/push.sh` command, right? it would be great if here has a comment that says about it.

Comment (danielnorberg) [inline circle.yml]: Yeah, and it should be `digdag/build` instead of `danieln/digdag-build`. Created a digdag docker hub account we can use for that.

**Comment by danielnorberg** [comment]
We cannot use oracle jdk for this build image. We'd have to use openjdk (e.g. zulu). Do we still want to go ahead with this?

**Pull request closed without merging**